src/sourmash/sig/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .__main__ import main
+from .__main__ import *         # bring all functions into top-level
```

Comment:
A bit nervous with this one, maybe also define `__all__` in `src/sourmash/sig/__main__.py`  to avoid unintended exports?

(not a blocker, but probably will trigger a hard-to-debug bug in the future 😅 )

Reply:
https://github.com/sourmash-bio/sourmash/issues/2714